bp-esa-frontend/src/components/filterSearchForm.js

```diff
@@ -0,0 +1,118 @@
+                            </label>
+                            </div>
+                            <div className="form-check">
+                            <label className="form-check-label">
+                                <input
+                                type="radio"
+                                className="form-check-input"
+                                name="filter"
+                                id="status"
+                                value="status"
+                                />
+                                Status
+                            </label>
+                            </div>
+                            <div className="form-check">
+                            <label className="form-check-label">
+                                <input
+                                type="radio"
+                                className="form-check-input"
+                                name="filter"
+                                id="addition"
+                                value="addition"
+                                />
+                                Addition
+                            </label>
+                            </div>
+                            <div className="form-check">
+                            <label className="form-check-label">
+                                <input
+                                type="radio"
+                                className="form-check-input"
+                                name="filter"
+                                id="removal"
+                                value="removal"
+                                />
+                                Removal
+                            </label>
+                            </div>
+                            <div className="form-check">
+                            <label className="form-check-label">
+                                <input
+                                type="radio"
+                                className="form-check-input"
+                                name="filter"
+                                id="channel"
+                                value="channel"
+                                />
+                                Channel
+                            </label>
+                            </div>
+                        </fieldset>
+                        </Col>
+                        <Col sm={9}>
+                        <div className="form-group">
+                            <input
+                            type="search"
+                            className="form-control"
+                            placeholder="&#xF002; Search"
+                            id="search"
+                            name="search"
+                            />
+                        </div>
+                        </Col>
+                    </Row>
+                </form>
+            </div>
+        );
+    }
```

Comment:
This can be simplified to one - 
```
<div className="form-check">
  <label className="form-check-label">
    <input
      type="radio"
       className="form-check-input"
        name="filter"
        id="channel"
         value="channel"
       />
    {filterCriteria}
  </label>
</div>
```
where {filterCriteria} is passed in via props in an array - that is ['Channel', 'Removal', 'Addition', etc]
The various criteria can then be passed to the html code with a map().